src/main/java/de/atextor/turtle/formatter/TurtleFormatter.java

```diff
@@ -952,4 +961,16 @@ public State write( final String content ) {
             return withLastCharacter( end ).withAlignment( alignment + content.length() );
         }
     }
+
+    public static void main(String[] args) throws IOException {
+        System.out.println(Arrays.stream(args).collect(Collectors.joining("\n")));
+        if (args.length != 1){
+            throw new IllegalArgumentException("usage: TurtleFormatter <file>");
+        }
+        String filename = args[0];
+        String content = Files.readString(Path.of(filename), StandardCharsets.UTF_8);
+        final FormattingStyle style = FormattingStyle.builder().alignPredicates(true).alignObjects(true).build();
+        final TurtleFormatter formatter = new TurtleFormatter(style);
+        System.out.println(formatter.applyToContent(content));
+    }
```

Comment:
Although I understand the reason and intention behind this, please remove the `main` method. TurtleFormatter is used as a library also in contexts where another `main` method is explicitly undesirable.
(But https://github.com/atextor/owl-cli/issues/20 will solve this problem! turtle-formatter will be a module in the overall command line tool, so you'll have an entry point for convenient local testing)